datetime_exercise.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+
+def get_date():
+    flag = True
+    while flag:
+        try:
```

Comment:
Always /manually-raising-throwinthrow the most specific exception - 
https://stackoverflow.com/questions/2052390g-an-exception-in-python
https://docs.python.org/3/library/exceptions.html

I don't think that you have to use try except in this ex